pageFactory/pageFactory.ts

```diff
@@ -0,0 +1,29 @@
+import { Page } from "@playwright/test";
+import { AddUserPage } from "../pages/addUserPage";
+import { EditUserPage } from "../pages/editUserPage";
+import { DeleteUserPage } from "../pages/deleteUserPage";
+import { HomePage } from "../pages/homePage";
+
+export class PageFactory {
+  public page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  getAddUserPage(): AddUserPage {
```

Comment:
as I said previously, it's okay for now, but add comment that we need to create some generic one , which will work for all pages.